operator/pkg/apis/istio/v1alpha1/values_types.pb.go

```diff
@@ -4456,307 +4464,308 @@ func init() {
-	0xd5, 0x3f, 0xe4, 0x00, 0xb6, 0x74, 0x2f, 0x30, 0x24, 0x5f, 0x02, 0x92, 0xfe, 0xb2, 0xb6, 0x13,
-	0x5d, 0x9f, 0xe1, 0xf3, 0xce, 0xe0, 0x62, 0x11, 0x40, 0x37, 0xac, 0xc0, 0x67, 0xb7, 0x5d, 0x6c,
-	0x53, 0x12, 0x88, 0xea, 0xb0, 0x20, 0x78, 0x79, 0x25, 0x9e, 0x18, 0xc6, 0xb6, 0xb6, 0xe3, 0x8d,
-	0x10, 0xf7, 0x67, 0x33, 0xaa, 0xdf, 0x02, 0x12, 0x55, 0x7d, 0x1a, 0xaf, 0xef, 0x94, 0x33, 0x4b,
-	0xab, 0x9e, 0x5c, 0x86, 0xea, 0x79, 0x17, 0x4a, 0x61, 0x9b, 0x9b, 0x6b, 0x39, 0xe2, 0x04, 0x50,
-	0xfd, 0x87, 0x09, 0x98, 0xe4, 0x03, 0xf0, 0xd0, 0x3d, 0xc8, 0x1b, 0x36, 0x91, 0xf7, 0x73, 0x2e,
-	0xf1, 0x5d, 0x7a, 0x50, 0xb2, 0x69, 0xd8, 0x04, 0x6d, 0xc0, 0xb4, 0x4c, 0x01, 0x05, 0x6e, 0x4d,
-	0xac, 0x1c, 0x20, 0xf9, 0xed, 0x88, 0x16, 0x52, 0xa2, 0x35, 0x98, 0x14, 0x4f, 0xdb, 0x32, 0x0b,
-	0xb1, 0x98, 0xae, 0x13, 0x0f, 0x6e, 0xaf, 0xa0, 0xe2, 0xd5, 0x47, 0x2c, 0xf0, 0x94, 0x35, 0x72,
-	0x0b, 0x99, 0x89, 0x7f, 0x4d, 0xd0, 0xa0, 0xa7, 0x4c, 0xcb, 0xca, 0x70, 0x5a, 0x6a, 0xd9, 0x5b,
-	0x19, 0x41, 0x7e, 0xa0, 0x59, 0x42, 0x5a, 0xf4, 0x15, 0x2c, 0x7a, 0x49, 0xcb, 0x2e, 0x4b, 0x97,
-	0xe5, 0xd3, 0x62, 0x4c, 0x4b, 0x65, 0x7a, 0x00, 0xda, 0x00, 0x76, 0xf4, 0x18, 0x0a, 0xe2, 0x73,
-	0x15, 0xb6, 0xa2, 0x73, 0x83, 0x57, 0x74, 0x9a, 0x53, 0x6d, 0xdb, 0x24, 0x51, 0x09, 0xbb, 0x90,
-	0xaa, 0x84, 0x5d, 0x86, 0x02, 0x3d, 0x0f, 0x3e, 0xcf, 0x16, 0x66, 0x26, 0x02, 0xa0, 0xa7, 0x00,
-	0x6d, 0xec, 0xc9, 0x0f, 0x54, 0xa4, 0x8b, 0x3e, 0xf0, 0xbd, 0x21, 0x46, 0x8a, 0x56, 0x61, 0xfc,
-	0x44, 0xf7, 0xb0, 0xf4, 0xbf, 0x63, 0xdf, 0xbb, 0x44, 0x77, 0x48, 0xe3, 0x14, 0xe8, 0x19, 0xcc,
-	0x90, 0xd8, 0x29, 0x94, 0xe5, 0x2b, 0xcb, 0xe9, 0x7a, 0xe2, 0xf8, 0x19, 0xd5, 0x12, 0x1c, 0xec,
-	0xc4, 0x06, 0xd3, 0x69, 0xea, 0x2d, 0xe6, 0x62, 0x33, 0x23, 0x96, 0x80, 0xb1, 0xd8, 0x32, 0xfc,
-	0x0e, 0x43, 0xae, 0xc4, 0xaa, 0x88, 0x2d, 0x53, 0x60, 0x55, 0x81, 0xc5, 0x6c, 0xdb, 0xa9, 0xde,
-	0x85, 0xb7, 0x2e, 0xf4, 0x5b, 0xd4, 0x45, 0x98, 0xcf, 0x7a, 0x4c, 0x54, 0x67, 0xa1, 0x92, 0x7a,
-	0xdc, 0x51, 0x7f, 0x0b, 0xa5, 0xc4, 0xb7, 0x7a, 0x3f, 0x72, 0xd9, 0x49, 0x05, 0x4a, 0x89, 0xbd,
-	0x79, 0xf0, 0xe5, 0x80, 0x77, 0x1c, 0x54, 0x81, 0xe2, 0x8b, 0xa3, 0x46, 0x7d, 0x77, 0x7b, 0x7f,
-	0x6f, 0x7f, 0x77, 0xa7, 0x7a, 0x03, 0x15, 0x61, 0x6a, 0x67, 0x77, 0xaf, 0xf6, 0xe2, 0xa0, 0x59,
-	0xcd, 0x21, 0x80, 0xc9, 0x46, 0x53, 0xdb, 0xdf, 0x6e, 0x56, 0xc7, 0xd0, 0x14, 0xe4, 0x9f, 0xef,
-	0xed, 0x55, 0xf3, 0x0f, 0x7e, 0x17, 0x04, 0x85, 0x0c, 0x2d, 0x2c, 0x62, 0xf5, 0x06, 0x2a, 0xc5,
-	0x0c, 0x71, 0x35, 0xc7, 0xc4, 0x48, 0x13, 0x5d, 0x1d, 0x63, 0x9d, 0xc4, 0x2c, 0x5f, 0x35, 0x8f,
-	0xe6, 0xa0, 0x42, 0x1d, 0x6c, 0x6f, 0x63, 0xdb, 0xeb, 0x78, 0xb5, 0x16, 0xb6, 0xfd, 0xea, 0x38,
-	0x9a, 0x86, 0x71, 0x9b, 0xda, 0xb8, 0x3a, 0xb1, 0xb5, 0xf8, 0xcf, 0x3f, 0xdc, 0xb9, 0xf1, 0x87,
-	0x1f, 0xee, 0xdc, 0xf8, 0xb7, 0x1f, 0xee, 0xdc, 0x78, 0x15, 0xfe, 0x33, 0x98, 0x93, 0x49, 0xbe,
-	0x16, 0x1f, 0xfe, 0x57, 0x00, 0x00, 0x00, 0xff, 0xff, 0xd9, 0xa3, 0xbc, 0x80, 0x4b, 0x46, 0x00,
-	0x00,
+	// 4836 bytes of a gzipped FileDescriptorProto
```

Comment:
This is no longer needed.

Reply:
this is generated, will be gone once the proto change is removed.